about.html

```diff
@@ -0,0 +1,125 @@
+            </ul>
+            <div class="hbuttoncontainer"><button class="headerbutton">News</button></div>
+          </nav>
+        </header>
+      <main>
+        <div class="a-headline">
+          <p class="a-ph1">"Stay in the game!"</p>
+          <h1  class="a-p1">
+            <span>Worldwide broadcast E3 2021
+          </h1>
+          <h2>
+            <span class="a-pd1">Worldwide broadcast E3 2021
+          </h2>
+          <p class="a-pheadline">
+            E3 is the world’s premier event for computer and video games and related products. 
+            The show is owned and operated by the ESA, the US trade association dedicated to serving 
+            the business and public affairs needs of the companies developing and publishing 
+            interactive games for video game consoles, handheld devices, personal computers, and the 
+            internet.
+          </p>
+          <p class="a-contact">Press Contacts: Kat Jones, Motiv PR, kjones@motivpr.com
+            and Johner Riehl, Wonacott PR, jriehl@wonacottpr.com</p>
+        </div>
+        <section class="info">
+          <h2 class="infotittle">Electronic Entertainment Expo Logo 2021</h2>
+          <h2 class="eventsline">____</h2>
+          <p class="infodescpt">The E3 logo and variations are trademarks and/or registered trademarks 
+            of the Entertainment Software Association.</p>
+          <div class="infocard"> <img class="logoimg" src="./imgs/E3Logo.png" alt=""></div>
+        </section>
+        <section class="pastinfo">
+          <h2 class="pastinfotittle">See Our Last Events</h2>
+          <h2 class="eventsline">____</h2>
+          <p class="infodescpt">Take a look to the last two years E3 events and all the historical announcements</p>
+          <div class="infobuttoncontainer">
+          <button class="lastinfobutton"><span class="infobuttontext">E3 2019</span><br>
+            <span class="infobuttontext1">2019 was the 25th E3 press conference</span></button>
+          <button class="lastinfobutton1"><span class="infobuttontext">E3 2020</span><br>
+            <span class="infobuttontext1">2020 was the 16th E3 press conference</span></button>
+          </div>
+        </section>
+        <section class="partner">
+          <div>
+          <h2 class="partnertittle">Partner</h2>
+          <h2 class="eventsline">____</h2>
+          </div>
+          <div class="partnerimgs">
+          <img class="partnerimg" src="./imgs/white-nvidia-logo-1png.png" alt="">
+          <img class="partnerimg" src="./imgs/SONY-SQUARE1.png" alt="">
+          <img class="partnerimg" src="./imgs/ubisoft-logo1.png" alt="">
+          <img class="partnerimg" src="./imgs/nintendologo1.png" alt="">
+          <img class="partnerimg" src="./imgs/Electronic_Arts-Logo.png" alt="">
+        </div>
+        </section>
+      </main>
+      <footer class="aboutfooter">
+        <img class="footerimg" src="./imgs/E3logowhite.png" alt="">
```

Comment:
- [ ] A good practice for `<img>` tag is always add a value for `alt` tag, please add it. 🐻